build-tools/timing/timing.targets

```diff
@@ -7,9 +7,18 @@
   <UsingTask AssemblyFile="..\..\bin\Build$(Configuration)\xa-prep-tasks.dll" TaskName="Xamarin.Android.BuildTools.PrepTasks.GitCommitHash" />
   <UsingTask AssemblyFile="..\..\bin\Build$(Configuration)\Xamarin.Android.Tools.BootstrapTasks.dll" TaskName="Xamarin.Android.Tools.BootstrapTasks.ProcessMSBuildTiming" />
   <Target Name="LogcatTiming">
-    <Exec Command="&quot;$(AdbToolPath)\$(AdbToolExe)&quot; $(_AdbTarget) $(AdbOptions) logcat -v threadtime -d > tmp-logcat-timing.log" />
+    <PropertyGroup>
+      <_DefaultLogcatInputFilename>tmp-logcat-timing.log</_DefaultLogcatInputFilename>
```

Comment:
Should this include a "random" filename element, e.g. via `Path.GetRandomFileName()`?

```xml
<_DefaultLogcatInputFilename>tmp-logcat-timing.log.$([System.IO.Path]::GetRandomFileName())</_DefaultLogcatInputFilename>
```

Additionally, should `$(_DefaultLogcatInputFilename)` ever be *deleted*?